crates/core/c8y_api/src/smartrest/operations.rs

```diff
@@ -218,6 +257,8 @@ impl Operation {
 #[serde(rename_all = "snake_case")]
 pub struct OnMessageExec {
     command: Option<String>,
+    command_name: Option<String>,
+    command_payload: Option<String>,
```

Comment:
question: these fields are meant to be for `.template` files, but right now it's technically possible for them to be in non-template files as well, right?